docker-compose/ansible/environments/docker/group_vars/all

```diff
@@ -0,0 +1,53 @@
+---
+
+whisk_version_name: local
+config_root_dir: /openwhisk
+whisk_logs_dir: /logs
+nginx_conf_dir: /etc/api-gateway/generated-conf.d
+docker_registry: ""
+docker_dns: ""
+bypass_pull_for_local_images: true
+
+db_prefix: "local_"
+
+db_provider: CouchDB
+db_port: 5984
+db_protocol: http
+db_username: whisk_admin
+db_password: some_passw0rd
+db_host: "{{ groups['db'] | first }}"
+
+# API GW connection configuration
+apigw_auth_user: ""
+apigw_auth_pwd: ""
+apigw_host_v2: "http://{{ groups['apigateway']|first }}:{{apigateway.port.api}}/v2"
+
+limits:
+  actions:
+    invokes:
+      perMinute: 60
+      concurrent: 30
+      concurrentInSystem: 5000
+  triggers:
+    fires:
+      perMinute: 60
+
+# Switch on runc
+invoker_use_runc: true
+
+catalog_repos:
+  openwhisk-catalog:
```

Comment:
is this configuration used ?

Reply:
I think the `init-catalog` in the Makefile has it defined static. If it's not used, can we remove it ?